lab1.js

```diff
@@ -68,6 +73,39 @@ var sentence1 = "More food please.",
  HINT: the "split" method on String will be useful.
 */
 
+function chirpMakerOne(sentence) {
+  sentence = sentence.split(" ");
+  for (var i = 0; i < sentence.length; i++) {
+    sentence[i] = "chirp";
+  }
+  sentence = sentence.join(" ") + ".";
+  console.log("The meerkat says " + sentence);
+  return sentence;
+}
+
+sentence1 = chirpMakerOne(sentence1);
+sentence2 = chirpMakerOne(sentence2);
+
+// Method 2
+// Reset sentence values
+var sentence1 = "More food please.",
```

Comment:
don't need to declare the variable again (use the var keyword), since you already did it in lines 65 and 66. In other words, you could take out the "var" and just assign sentence 1 and sentence 2 these values.